db/migrate/20180123140634_remove_limit_ldap_filter.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveLimitLdapFilter < ActiveRecord::Migration[5.1]
+  def change
+    change_column 'auth_sources', :ldap_filter, :text, :limit => nil
```

Comment:
This migration is irreversible, you need to make an up/down migration (since it doesn't know what the old limit is)